.github/workflows/build.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+# https://github.com/marketplace/actions/electron-builder-action
+
+on: push
+
+jobs:
+  release:
+    name: Test and Build
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+
+    env:
+      XDG_CACHE_HOME: .cache
+      ELECTRON_CACHE: .cache/electron
+      ELECTRON_BUILDER_CACHE: .cache/electron-builder
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v1
+
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Cache bigger downloads
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: .cache
+          key: ${{ runner.os }}-${{ hashFiles('package.json', 'package-lock.json', 'electron-builder.yml') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ hashFiles('package.json', 'package-lock.json', 'electron-builder.yml') }}
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: npm ci --prefer-offline --no-audit --progress=false --cache .cache/npm
+
+      - name: Build
+        run: npm run build
+
+      - name: Test
+        run: npm run test
+
+      - name: Test end-to-end
+        # TODO if: !startsWith(matrix.os, 'ubuntu') # xvfb slow on gh, we run test:e2e on Travis for now
```

Comment:
Is this something we can check in the test:e2e? Is it only slow for gh or is it also slow for ubuntu users?

Reply:
Unsure why test:e2e fails on Linux at GH. The same code runs fine on Linux on Travis, so I assume something is wrong with the way we run xvfb (we spawn it separately there)